docs/wordpress-security.md

```diff
@@ -0,0 +1,29 @@
+Any self-hosted software comes with security obligations. Especially so with WordPress, which as the most popular CMS on the web is highly targeted with hack attempts.
+
+As Next.js serves your front-end content, and in our configuration does most if not all requests server-side, most automated attacks won't know where to look. They will attempt a request to yourdomain.com/wp-login.php fail, then they'll go away.
+
+But we still need to secure the graphql connection, and the WP admin interface.
+
+## Keeping software up to date
+
+Managed WordPress from WP Engine or Pantheon for example will go a long way by keeping your production install. As WordPress specialized hosts, they also have expertise at [taking steps to lock WordPress down](https://wpengine.com/blog/11-top-wordpress-security-concerns-how-wp-engine-takes-care-of-them-for-you/).
+
+## Securing Graphql
+
+We're still figuring out what we want to implement in our bubs opinionated default.
+
+Here are some resources we're reviewing:
+
+- https://www.wpgraphql.com/docs/authentication-and-authorization/
+- https://github.com/wp-graphql/wp-graphql/blob/develop/docs/security.md
+- https://github.com/wp-graphql/wp-graphql/issues/1896
+
+## WordPress logins and Google Auth
+
+To help lock down WordPress, we typically will enable and configure the [Google Apps Login](https://wordpress.org/plugins/google-apps-login/) plugin. There are two optional configurations in bubs which are disabled by default, but which can be enabled to add extra security.
+
+1. In `wordpress/functions.php`, you can enable `google-login-force.php`. In production, this will redirect logged out traffic to a google login prompt. This essentially disables email/password logins. Only enable if 100% of login emails have google accounts at the same email.
```

Comment:
Technically the wp-login.php script can still be POSTED to, which is what I believe a lot of attacks do directly. I wouldn't say it disables email/password logins, it just makes it very difficult for humans to do them.